src/compoenets/renderContinent.js

```diff
@@ -0,0 +1,13 @@
+import { React, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { loadAPIData } from '../redux/continent/continent';
+
+const CovidDataRender = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loadAPIData());
```

Comment:
- [ ] I can see that the Data is not being rendered properly on the Detail page, Let's make sure APIs are working fine with the app. Let's make sure we use the API that shows details for most of the companies.